lib/matplotlib/backends/backend_cairo.py

```diff
@@ -39,6 +21,27 @@
 from matplotlib.transforms import Affine2D
 
 
+def import_cairo():
```

Comment:
Would leave this at the toplevel instead of wrapping it in a function; in any case it shouldn't be a public function at least.